Dockerfile

```diff
@@ -1,45 +1,57 @@
 FROM ruby:2.3
 
-RUN apt-get update && \
-    apt-get install --no-install-recommends -y cron && \
-    rm -rf /var/lib/apt/lists/* && \
-    apt-get clean
+RUN supercronicUrl=https://github.com/aptible/supercronic/releases/download/v0.1.3/supercronic-linux-amd64 && \
+    supercronicBin=/usr/local/bin/supercronic && \
+    supercronicSha1sum=96960ba3207756bb01e6892c978264e5362e117e && \
+    curl -fsSL -o "$supercronicBin" "$supercronicUrl" && \
+    echo "$supercronicSha1sum  $supercronicBin" | sha1sum -c - && \
+    chmod +x "$supercronicBin"
 
 ENV RAILS_ENV=production \
+    RAILS_LOG_TO_STDOUT=true \
     RAILS_SERVE_STATIC_FILES=true
 
 WORKDIR /usr/src/app
+
 COPY . ./
-RUN echo $SOURCE_COMMIT > REVISION
 
 RUN buildDeps='libmagic-dev' && \
     apt-get update && \
     apt-get install --no-install-recommends -y $buildDeps && \
-    rm -rf /var/lib/apt/lists/* && \
-    bundle install --deployment --without development test && \
+    echo 'gem: --no-document' >> ~/.gemrc && \
+    bundle config build.nokogiri "--use-system-libraries" && \
+    bundle install --deployment --without development test -j 4 && \
     apt-get purge -y --auto-remove $buildDeps && \
-    echo "Foodsoft::Application.config.secret_token = ENV['SECRET_KEY_BASE']" > config/initializers/secret_token.rb && \
-    mkdir -p log && \
-    ln -sfn /dev/stdout log/production.log && \
-    bundle exec whenever --update-crontab
+    rm -Rf /var/lib/apt/lists/* /var/cache/apt/* ~/.gemrc ~/.bundle
+
+# use the default config (or else the app won't start)
+# most non-system-related settings can be changed in the app anyway
+RUN mv config/app_config.yml.SAMPLE config/app_config.yml
```

Comment:
i do not like to have a `app_config.yml` in the image. That's a "mandatory configuration" which must be supplied by the user, which can be done with a two line `Dockerfile` and I expect that the application won't start if the config is missing. We could add a nice error message to improve usability.

Reply:
I'll add an error message. Could this also be supplied with a docker volume?
Needs to end up in the docker docs as well.

Reply:
if you provide an ABSOLUTE path for it on the host side it should work.

Reply:
Thanks, I'll experiment with it and mention it in the docs.